deploy/backplane/acs/01-acs-admins-project.ClusterRole.yaml

```diff
@@ -35,14 +36,17 @@ rules:
   - ""
   resources:
   - pods
+  - exec
```

Comment:
Why is this a day-to-day permission and not break-glass?

Reply:
@Nanyte25 I don't think this is required